core/domain/classifier_domain_test.py

```diff
@@ -108,6 +108,42 @@ def test_to_dict(self):
             expected_training_job_dict,
             observed_training_job.to_dict())
 
+    def test_to_player_dict(self):
+        training_job_dict = {
+            'job_id': 'exp_id1.SOME_RANDOM_STRING',
+            'algorithm_id': 'TextClassifier',
+            'interaction_id': 'TextInput',
+            'exp_id': 'exp_id1',
+            'exp_version': 1,
+            'next_scheduled_check_time':
+                datetime.datetime.strptime(
+                    '2017-08-11 12:42:31', '%Y-%m-%d %H:%M:%S'),
+            'state_name': 'a state name',
+            'status': 'NEW',
+            'training_data': [
+                {
+                    'answer_group_index': 1,
+                    'answers': ['a1', 'a2']
+                },
+                {
+                    'answer_group_index': 2,
+                    'answers': ['a2', 'a3']
+                }
+            ],
+            'classifier_data': {},
+            'data_schema_version': 1
+        }
+        expected_training_job_dict = {
```

Comment:
Consider renaming to expected_training_job_player_dict (since we have two different concepts here, a "dict" and a "player dict").